service/controller/app/v1/resource/chart/create.go

```diff
@@ -0,0 +1,28 @@
+package chart
+
+import (
+	"context"
+	"fmt"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
+	customResource, err := key.ToCustomResource(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+	name := key.AppName(customResource)
+	chart, err := r.g8sClient.ApplicationV1alpha1().Charts(r.watchNamespace).Get(name, v1.GetOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("chart %#q is not created yet", name))
```

Comment:
Please have a look at how we prefer to log information: https://github.com/giantswarm/fmt/blob/master/go/log_messages.md.

Reply:
this log will be deleted.

Reply:
I think the logging makes sense, just the wording is off and it should align with the rest. So far we have logging before and after an action to have a better understanding of the internals at runtime. We can hang if this is unclear.

Reply:
okay, then it's contradicted with @rossf7 but I also think logging in here make sense.